packages/core/echo/echo-schema/src/automerge/automerge-object.ts

```diff
@@ -134,7 +134,9 @@ export class AutomergeObject implements TypedObjectProperties {
     return this._id;
   }
 
-  [base] = this as any;
+  get [base]() {
```

Comment:
Fixes test matcher. It's making a deep object copy, not copying symbols but setting a prototype making getters accessible.